install/kubernetes/helm/istio/charts/prometheus/values.yaml

```diff
@@ -31,8 +31,11 @@ tolerations: []
 podAntiAffinityLabelSelector: []
 podAntiAffinityTermLabelSelector: []
 
-# Controls the frequency of prometheus scraping
-scrapeInterval: 15s
```

Comment:
This will be backwards incompatible, I am not sure we can break existing users like this